### PR TITLE
Displaying Pokemon Data from API

### LINKED ISSUE #9: [3pt]  - Create feature: add  new like - Student A
When the user clicks on the Like button of an item (on Homepage), the interaction is recorded in the Involvement API and the screen is updated.

### LINKED ISSUE #12: [4pt]  - Display number of likes for each item on the Homepage - Student A
When the page loads, the webapp the [Involvement API](https://www.notion.so/microverse/Involvement-API-869e60b5ad104603aa6db59e08150270) to show the item likes and combines them with the data from the base API.

Remember that your page should make only 2 requests:
- one to the base API
- and **one** to the Involvement API.

This task **does not** include displaying the likes button (heart icon on the wireframe) for each item.

### LINKED ISSUE #15: [4pt]  - Display list of items on the Homepage - Student A
Create the main part of the homepage that keeps the layout from the wireframe: 

Screenshot 2021-06-27 at 21 50 23

When the page loads, the webapp retrieves data from  the selected API and shows the list of items on screen with the basic data (e.g. title + image).

Prepare also "Comments" and "Reservations" buttons. They should be doing nothing - just being displayed.

This task **does not** include displaying the number of likes for each item.

This task **does not** include the counter of the items.

### PR DESCRIPTION
**Pull request by JuanDiegoGuerra**
### Rendering API Data

- Use of APIs for:
   - Display Pokemon data
   - Display Pokemons' Likes

### CODE REVIEW

**Review by lincoln1883 — COMMENTED** [review]
Hey, @JuanDiegoGuerra good job so far on implementing the Api connections.

File: src/modules/api_data.js
Comment (lincoln1883) [inline src/modules/api_data.js:1]: A recommendation from my side would be to use one file for the API calls and export them individually.

**Review by lincoln1883 — CHANGES_REQUESTED** [review]
Please make the necessary changes. 👍🏾

**Comment by JuanDiegoGuerra** [comment]
###  Required changes made 🛠
**All linters are enable!**
- Fixed _"use one file for the API calls and export them individually"_ ✅
  - Now the app is adding 'likes' correctly!

todolist1

**Review by lincoln1883 — APPROVED** [review]
Hello Juan good job on implementing the requested changes, you are free to merge your changes.

**Pull request merged**